Common/src/CConfig.cpp

```diff
@@ -1150,7 +1150,9 @@ void CConfig::SetConfig_Options() {
 
   /*!\par CONFIG_CATEGORY: FluidModel \ingroup Config*/
   /*!\brief FLUID_MODEL \n DESCRIPTION: Fluid model \n OPTIONS: See \link FluidModel_Map \endlink \n DEFAULT: STANDARD_AIR \ingroup Config*/
-  addEnumOption("FLUID_MODEL", Kind_FluidModel, FluidModel_Map, STANDARD_AIR);
+  addEnumOption("FLUID_MODEL", Kind_FluidModel, FluidModel_Map, COOLPROP);
+  /*!\brief FLUID_NAME \n DESCRIPTION: Fluid name \n OPTIONS: see coolprop homepage \n DEFAULT: nitrogen \ingroup Config*/
+  addStringOption("FLUID_NAME", FLUID_NAME, string("nitrogen"));
```

Comment:
I wonder if there is a way to combine this with the Mutation++  options.

Reply:
That is a good idea. 
I will try to finish this new feature as soon, and then if it is possible, I am happy to work with others on this combination.